app/src/main/java/ar/com/wolox/android/cookbook/room/dialog/RoomInputDialog.kt

```diff
@@ -0,0 +1,55 @@
+package ar.com.wolox.android.cookbook.room.dialog
+
+import android.app.AlertDialog
+import android.content.Context
+import android.text.InputFilter
+import android.text.InputType
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+import ar.com.wolox.android.cookbook.R
+import javax.inject.Inject
+
+class RoomInputDialog @Inject constructor() {
+
+    private lateinit var input: EditText
+
+    fun showDialog(context: Context, title: Int, listener: RoomInputDialogListener) = AlertDialog.Builder(context).apply {
+
+        input = EditText(context)
+        input.apply {
```

Comment:
Same as above. On this case it's okey to use `apply`, but on the previous line. `input = EditText(context).apply { ... }`